packages/flutter_tools/lib/src/resident_runner.dart

```diff
@@ -198,15 +198,14 @@ class FlutterDevice {
     if (flutterViews.any((FlutterView view) {
       return view != null &&
              view.uiIsolate != null &&
-             view.uiIsolate.pauseEvent.isPauseEvent;
+             view.uiIsolate.pauseEvent?.isPauseEvent == true;
```

Comment:
Alternatively you can maintain the invariant mentioned in this comment:

https://github.com/flutter/flutter/blob/master/packages/flutter_tools/lib/src/vmservice.dart#L1091

by rewriting this line:

https://github.com/flutter/flutter/blob/master/packages/flutter_tools/lib/src/vmservice.dart#L1171

as something like:
```dart
pauseEvent = map['pauseEvent'] ?? // resume event or a new sentinel event kind. 
```

and/or update the comment on Isolate's pauseEvent field.

Reply:
But that would mean testing the vmservice :) ... I'll give it a shot.